lib/kino/explorer.ex

```diff
@@ -147,30 +147,33 @@ defmodule Kino.Explorer do
         grouped = (column in groups) |> to_string(),
         nulls = Series.nil_count(series) |> to_string(),
         into: %{} do
-      if summary_type == :numeric do
-        mean = Series.mean(series)
-        mean = if is_float(mean), do: Float.round(mean, 2) |> to_string(), else: to_string(mean)
-        min = Series.min(series) |> to_string()
-        max = Series.max(series) |> to_string()
-        keys = ["min", "max", "mean", "nulls"]
-        values = [min, max, mean, nulls]
-
-        keys = if has_groups, do: keys ++ ["grouped"], else: keys
-        values = if has_groups, do: values ++ [grouped], else: values
-
-        {column, %{keys: keys, values: values}}
-      else
-        %{"counts" => top_freq, "values" => top} = most_frequent(series)
-        top_freq = top_freq |> List.first() |> to_string()
-        top = List.first(top) |> to_string()
-        unique = count_unique(series)
-        keys = ["unique", "top", "top freq", "nulls"]
-        values = [unique, top, top_freq, nulls]
-
-        keys = if has_groups, do: keys ++ ["grouped"], else: keys
-        values = if has_groups, do: values ++ [grouped], else: values
-
-        {column, %{keys: keys, values: values}}
+      cond do
+        summary_type == :numeric -> 
+          mean = Series.mean(series)
+          mean = if is_float(mean), do: Float.round(mean, 2) |> to_string(), else: to_string(mean)
+          min = Series.min(series) |> to_string()
+          max = Series.max(series) |> to_string()
+          keys = ["min", "max", "mean", "nulls"]
+          values = [min, max, mean, nulls]
+
+          keys = if has_groups, do: keys ++ ["grouped"], else: keys
+          values = if has_groups, do: values ++ [grouped], else: values
+
+          {column, %{keys: keys, values: values}}
+        summary_type == :categorical and compute_frequencies?(series) ->
```

Comment:
Since we're ignoring the entire summary in unsupported cases (and we should), let's call it `compute_summaries?`

Reply:
i looked at the func `most_frequent/1` for which this is being created and it was calling `Series.frequencies/1` - hence named it `compute_frequencies` rather than naming generic `compute_summaries`.

Reply:
The point is that by returning `false`, we're ignoring not just `most_frequent`, but the entire summary.
Either way, `count_unique` gives us the same error.